crates/aggkit-prover/src/cli/mod.rs

```diff
@@ -0,0 +1,35 @@
+use std::path::PathBuf;
+
+use clap::{Parser, Subcommand, ValueHint};
+
+use crate::version;
+
+/// Aggkit prover command line interface.
+#[derive(Parser)]
+#[command(version = version())]
+#[command(propagate_version = true)]
+pub struct Cli {
+    #[command(subcommand)]
+    pub cmd: Commands,
+}
+
+#[derive(Subcommand)]
+pub enum Commands {
+    Run {
+        /// The path to the configuration file.
+        #[arg(long, short, value_hint = ValueHint::FilePath, default_value = "aggkit-prover.toml", env = "CONFIG_PATH")]
+        config_path: PathBuf,
+    },
+
+    Config,
+
+    ValidateConfig {
+        /// The path to the aggkit-prover configuration file.
+        #[arg(
+            value_hint = ValueHint::DirPath,
+        )]
+        config_path: PathBuf,
```

Comment:
Is this a directory or a file? The hint does not agree with the comment.

Also a nitL does the `#[arg(..)]` attribute need to be over three lines?

Reply:
It is the path to the configuration file